csharp/build_release.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+cd $(dirname $(readlink $BASH_SOURCE))
+
+#Disable some unwanted dotnet options
```

Comment:
Tiny nit-pick: I would suggest having a space after the `#` in each comment line, for readability. (Having looked at a couple of shell scripts in the root directory, it looks like that's common in this repo too.)

Reply:
Fixed!